### PR TITLE
Update GLSL_EXT_ray_query.txt

### PR DESCRIPTION
**Pull request by TylerNowicki**
Clarify rayQueryGetIntersectionCandidateAABBOpaqueEXT description

### CODE REVIEW

File: extensions/ext/GLSL_EXT_ray_query.txt [hunk 1]
```diff
@@ -442,8 +442,8 @@ Additions to Chapter 8 of the OpenGL Shading Language Specification
 
         bool rayQueryGetIntersectionCandidateAABBOpaqueEXT(rayQueryEXT q);
 
-    Returns whether current candidate intersection is opaque geometry
-    or not.
+    Returns whether current candidate intersection is opaque AABB or
```
Comment (gnl21) [inline extensions/ext/GLSL_EXT_ray_query.txt:445]:
```suggestion
    Returns whether current candidate intersection is an opaque AABB or
```

**Comment by fluppeteer** [comment]
I know this is pedantic, but I do have to look twice at anything that says "returns whether something is true or not" to confirm it's not "returns whether A is true or B is true" for independent A and B. I probably hang around too many people who use boolean logic answers to either/or questions.

Could we remove the "or not"s, or say "returns true if the current candidate intersection is opaque"?

**Comment by dgkoch** [comment]
We should also update the changelog at the bottom and version at the top of the file.

**Review by dgkoch — APPROVED** [review]
LGTM - Thanks!

**Pull request merged**